### ISSUE TITLE
warning: URI.escape is obsolete

### ISSUE THREAD

**Issue by gionn** [sous-chefs/docker#1138]
## :ghost: Brief Description

It looks like that the docker-api version needs to be bumped to 2.0.0 to avoid spurious deprecation warnings when running this cookbook on recent Chef-Infra versions.

## :pancakes: Cookbook version

7.0.0

## :woman_cook:  Chef-Infra Version

16.6.14

## :tophat: Platform details

Ubuntu 18.04

## Steps To Reproduce

Just run the example recipe

## :police_car: Expected behavior

No warnings printed

## :heavy_plus_sign: Additional context

```
/opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/container.rb:346: warning: URI.escape is obsolete
[dev-eng] out: /opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/image.rb:129: warning: URI.escape is obsolete
[dev-eng] out: /opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/container.rb:346: warning: URI.escape is obsolete
[dev-eng] out: /opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/image.rb:129: warning: URI.escape is obsolete
[dev-eng] out: /opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/container.rb:346: warning: URI.escape is obsolete
[dev-eng] out: /opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/container.rb:346: warning: URI.escape is obsolete
[dev-eng] out: /opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/container.rb:346: warning: URI.escape is obsolete
[dev-eng] out: /opt/chef/embedded/lib/ruby/gems/2.7.0/gems/docker-api-1.34.2/lib/docker/image.rb:129: warning: URI.escape is obsolete
```

**Comment by ramereth** [comment]
@gionn This should get resolved once #1136 has been merged and release which should happen soon. Can you please verify that's the case?

**Comment by gionn** [comment]
I confirm that while that branch warnings are gone. Thanks!
